libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -38,8 +38,22 @@
 #include <utility>
 #include <vector>
 
-namespace pika::mpi::experimental {
+#if __has_include(<mpi-ext.h>)
+# include <mpi-ext.h>
+# ifndef OMPI_HAVE_MPI_EXT_CONTINUE
+#  pragma warning "MPIX Continuations not found in <mpi-ext.h>"
```

Comment:
Can we make this only a message or even not print it? I wouldn't want a `-Werror` build to error out on this not being available.

Alternatively, make `PIKA_WITH_MPI_CONTINUATIONS` an explicit CMake option, and the build will fail MPI continuations aren't available?

In either case, we could probably remove some of the members if continuations are disabled?

Reply:
I use this just as a sanity check when building on various machines in case the openmpi is somehow wring. it is useful to see the message, but I can add a cmake option.